Makefile

```diff
@@ -44,21 +44,16 @@ migrate-down:
 	@npm run migrate down --prefix server
 
 
-copy-sample-data:
-	@echo "++\n Copying server/sample-data to server/scripts/xlsx\n++"
-	@mkdir -p server/scripts/xlsx
-	@cp server/test-data/* server/scripts/xlsx
-
 seed-participants:
-	@echo "Seeding participants from server/scripts/xlsx"
+	@echo "Seeding participants from server/test-data"
 	@docker-compose -f docker-compose.dev.yml exec server npm run feed-participants $$SHEET
 
 seed-sites:
-	@echo "Seeding sites from server/scripts/xlsx"
+	@echo "Seeding sites from server/test-data"
 	@docker-compose -f docker-compose.dev.yml exec server npm run feed-sites $$SHEET
 
```

Comment:
Do we still need additional commands for seeding sites/participants, or can this all be accomplished by `seed-data`?

Reply:
Oh i think Noel already asked the same thing. Disregard.